features/zoeken/met-geslachtsnaam-en-geboortedatum/overzicht.feature

```diff
@@ -0,0 +1,226 @@
+
+    Abstract Scenario: de geslachtsnaam, voornamen en geboortedatum van personen komen exact overeen met de opgegeven geslachtsnaam, voornamen en geboortedatum
+      Gegeven de persoon 'Pieter' met burgerservicenummer '000000024'
+      * heeft de volgende gegevens
+        | geboortedatum (03.10) | geslachtsnaam (02.40) | voornamen (02.10) |
+        |              19830526 | Maassen               | Pieter            |
+      En de persoon 'Jan' met burgerservicenummer '000000025'
+      * heeft de volgende gegevens
+        | geboortedatum (03.10) | geslachtsnaam (02.40) | voornamen (02.10) | voorvoegsel (02.30) |
+        |              19830526 | Maassen               | Jan Peter         | van                 |
+      Als 'burgerservicenummer' wordt gevraagd van personen gezocht met geslachtsnaam '<geslachtsnaam>' en geboortedatum '1983-05-26' en parameters
+        | voornamen   |
+        | <voornamen> |
+      Dan wordt alleen '<persoon aanduiding>' gevonden
+
+      Voorbeelden:
+        | geslachtsnaam | voornamen | persoon aanduiding |
+        | Maassen       | Pieter    | Pieter             |
+        | maassen       | PIETER    | Pieter             |
+        | MAASSEN       | pieter    | Pieter             |
+        | maassen       | jan peter | Jan                |
+
+    Abstract Scenario: de geslachtsnaam, voorvoegsels en geboortedatum van personen komen exact overeen met de opgegeven geslachtsnaam, voorvoegsels en geboortedatum
+      Gegeven de persoon 'Pieter' met burgerservicenummer '000000024'
+      * heeft de volgende gegevens
+        | geboortedatum (03.10) | geslachtsnaam (02.40) | voornamen (02.10) |
+        |              19830526 | Maassen               | Pieter            |
+      En de persoon 'Jan' met burgerservicenummer '000000025'
+      * heeft de volgende gegevens
+        | geboortedatum (03.10) | geslachtsnaam (02.40) | voornamen (02.10) | voorvoegsel (02.30) |
+        |              19830526 | Maassen               | Jan Peter         | van                 |
+      Als 'burgerservicenummer' wordt gevraagd van personen gezocht met geslachtsnaam '<geslachtsnaam>' en geboortedatum '1983-05-26' en parameters
+        | voorvoegsel    |
+        | <voorvoegsels> |
+      Dan wordt alleen 'Jan' gevonden
+
+      Voorbeelden:
+        | geslachtsnaam | voorvoegsels |
+        | Maassen       | van          |
+        | maassen       | VAN          |
+        | MAASSEN       | Van          |
+
+  Regel: de optionele 'geslacht' parameter (niet hoofdlettergevoelig) kan worden gebruikt om de zoek criteria aan te scherpen
+
+    Abstract Scenario: geslachtsnaam, geslacht en geboortedatum van personen komen exact overeen met de opgegeven geslachtsnaam, geslacht en geboortedatum
+      Gegeven de persoon 'Aedel' met burgerservicenummer '000000027'
+      * heeft de volgende gegevens
+        | geboortedatum (03.10) | geslachtsaanduiding (04.10) |
```

Comment:
je zou ook de tabel kunnen vervangen door de stappen:
* is geboren op 16-05-1983
* is een vrouw

Reply:
er is nu een stap definitie waarmee het in één regel kan